<?xml version="1.0" ?>
<launch>
	<node pkg="handover" type="handover_server.py" name="handover_server" output="screen"/>
</launch>